README.md

```diff
@@ -27,14 +27,10 @@ which has been tested to use from this exporter:
  - `0.10.0.1`
  - `0.9.0.1`
 
-Build
-=====
+Build & install
```

Comment:
Ah, sorry. I see. I actually always start developing Go projects by doing something like
```sh
$ go get github.com/kawamuray/prometheus-kafka-consumer-group-exporter
$ cd $GOPATH/srv/github.com/kawamuray/prometheus-kafka-consumer-group-exporter
$ git checkout -b my-new-feature
```
...since the `go get` essentially is a `git clone`. I respect we all have different approaches 😄 , but I just felt like
```
$ mkdir -p $GOPATH/src/github.com/kawamuray
$ cd $GOPATH/src/github.com/kawamuray
$ git clone https://github.com/kawamuray/prometheus-kafka-consumer-group-exporter.git
$ cd prometheus-kafka-consumer-group-exporter
$ go get . && go build -o kafka_consumer_group_exporter .
```
was more work than necessary.

How about you make a pull request either to this branch with a proposal or create a brand new PR that I can review?